### ISSUE TITLE
Is it basically like dirbuster?

### ISSUE THREAD

**Issue by fannyhasbi** [wkdwilliams/LewScanner#2]

**Comment by wkdwilliams** [comment]
Exactly. I need to improve the code of this as I've seen some false positives on different servers and frameworks. For example, I tested on my Laravel app and was for some reason getting HTTP 300 (redirect) instead of giving 200 (OK).
